### DISCUSSION TITLE
(In  every file )Parsing Error: Invalid value for lib provided: es2022 eslint

### DISCUSSION THREAD

**Discussion by MihaelPavlov** [angular-eslint/angular-eslint#1696] [Q&A]
Hey i've been fighting from 1 week to find the problem in this eslint config file. Will appriciate the help from anybody or any suggestions if you have.

I think the problem is in any of the plugins in the eslintrc.json file.
Because if i remove the angular and ts plugins and restart the eslint, the error doesn't appear at all.

.eslintrc.json file -> https://pastebin.com/90LMNZL8

I am sharing and the package.json file with the dependencies 
And i need to fix it in Angular 16 

` 

"dependencies": {
    "@angular/animations": "16.0.0",
    "@angular/cdk": "16.0.1",
    "@angular/common": "16.0.0",
    "@angular/compiler": "16.0.0",
    "@angular/core": "16.0.0",
    "@angular/forms": "16.0.0",
    "@angular/material": "16.0.1",
    "@angular/material-moment-adapter": "16.0.1",
    "@angular/platform-browser": "16.0.0",
    "@angular/platform-browser-dynamic": "16.0.0",
    "@angular/router": "16.0.0",
    "@ng-select/ng-select": "11.0.0",
    "@ngneat/transloco": "^4.2.2",
    "chart.js": "^4.4.0",
    "crypto-js": "^4.1.1",
    "date-fns": "^2.29.3",
    "memoize-one": "^6.0.0",
    "moment": "^2.29.4",
    "ng-for-track-by-property": "16.0.1",
    "ng2-charts": "^5.0.3",
    "ngx-quill": "22.0.0",
    "rxjs": "~7.5.0",
    "tslib": "^2.3.0",
    "zone.js": "0.13.0"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "16.0.0",
    "@angular-devkit/core": "16.0.0",
    "@angular-devkit/schematics": "16.0.0",
    "@angular-eslint/builder": "16.0.1",
    "@angular-eslint/eslint-plugin": "16.0.1",
    "@angular-eslint/eslint-plugin-template": "16.0.1",
    "@angular-eslint/schematics": "16.0.1",
    "@angular-eslint/template-parser": "16.0.1",
    "@angular/cli": "16.0.0",
    "@angular/compiler-cli": "16.0.0",
    "@nx/angular": "16.1.4",
    "@nx/jest": "16.1.4",
    "@nx/js": "16.1.4",
    "@nx/workspace": "16.1.4",
    "@schematics/angular": "^14.2.8",
    "@types/crypto-js": "^4.1.1",
    "@types/jest": "^29.5.1",
    "@types/node": "20.1.0",
    "@types/quill": "^1.3.10",
    "@typescript-eslint/eslint-plugin": "5.11.0",
    "@typescript-eslint/parser": "5.11.0",
    "eslint": "^8.32.0",
    "husky": "^7.0.0",
    "jest": "29.5.0",
    "jest-environment-jsdom": "29.5.0",
    "jest-preset-angular": "13.1.1",
    "lint-staged": "^12.3.7",
    "ng-packagr": "16.0.0",
    "nx": "16.1.4",
    "nx-cloud": "16.0.5",
    "postcss-preset-env": "^8.3.2",
    "quill": "^1.3.7",
    "ts-jest": "29.1.0",
    "ts-node": "10.9.1",
    "tslib": "^2.4.1",
    "typescript": "4.9.5"
  },
`

**Comment by json-derulo** [comment]
Did you check [this StackOverflow question](https://stackoverflow.com/q/73119234/5470544)?
